src/backend/base/langflow/services/database/service.py

```diff
@@ -116,21 +135,32 @@ def _create_engine(self) -> AsyncEngine:
         # if the user specifies an empty dict, we allow it.
         kwargs = self._build_connection_kwargs()
 
+        poolclass_key = kwargs.get("poolclass")
+        default_poolclass = AsyncAdaptedQueuePool if url_components[0].startswith("sqlite") else NullPool
```

Comment:
`AsyncAdaptedQueuePool` is now the default pool for sqlite.
We should pass `None` and let sqlalchemy decide the default.